clinica/pipelines/t1_linear/info.json

```diff
@@ -0,0 +1,12 @@
+{
+    "id": "aramislab/t1-linear",
+    "author": "John Doe",
+    "version": "0.1.0",
+    "dependencies": [
+        {
+            "type": "binary",
+            "name": "clinica",
+            "version": ">=0.1.0"
+        }
+    ]
+}
```

Comment:
Your `info.json` should look like
```JSON
{
    "id": "aramislab/t1-linear",
    "author": " Mauricio Diaz",
    "version": "0.1.0",
    "space_caps": "42M",
    "space_wd": "42M",
    "dependencies": [
        {
            "type": "software",
            "name": "ants",
            "version": ">=2.3.1"
        }
    ]
} 
```
space_caps/space_caps were added recently: this is the size of the outputs for a single (participant, session). So replace these default values by the adequate one.